ldap_status.sh

```diff
@@ -1,32 +1,35 @@
 #!/bin/bash
 
 # LDAP username
-user=""
+USER=""
 
 # LDAP password
-password=""
+PASSWORD=""
 
 # Base dn of LDAP tree
-basedn=""
+BASEDN=""
 
 # eduPersonPrincipalName which will be searched
-searchedIdentity=""
+IDENTITY=""
 
-# List of LDPA hostnames separated by space
+# List of LDPA HOSTNAMES separated by space
 # Included ldap:// or ldaps://
```

Comment:
I think this wording is a bit confusing. I.e., it seems a bit unclear to me if I should add the ldaps:// or not. Perhaps change the text as well?

Reply:
Rewrited.